bazaarvoice/emodb
# [PD-257044] Databus Changes

## Jira Issue #

[PD-257044](https://bazaarvoice.atlassian.net/browse/PD-257044)

## What Are We Doing Here?

- Made changes to push to kafka instead of writing to databus
- Seperate API to handle write to databus

## How to Test and Verify

1. Check out this PR
2. Run mvn clean install

## Risk

### Level 

`Medium`

### Required Testing

`Smoke`

### Risk Summary

Smoke test on connectivity and others are required.

## Code Review Checklist

build and run.



[PD-257044]: https://bazaarvoice.atlassian.net/browse/PD-257044?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
